MULTIPLATFORM.md

```diff
@@ -0,0 +1,244 @@
+
+    apply plugin: 'kotlin-platform-common'
+
+    // Specify a group and a version of the library to access it in Android Studio.
+    group = 'org.greeting'
+    version = 1.0
+
+    dependencies {
+        // Set up compilation dependency on common Kotlin stdlib
+        compile "org.jetbrains.kotlin:kotlin-stdlib-common:$kotlin_version"
+    }
+
+#### 2.2 Android subproject
+
+The `android` subproject contains platform-dependent implementations of `expect`-declarations we've created in the
+`common` project. Let's implement our `expect`-class:
+
+    // greeting/android/src/main/kotlin/android.kt
+    package org.greeting
+
+    actual class Platform actual constructor() {
+        actual val platform: String = "Android"
+    }
+
+We build this project into a Java-library which can be used by Android Studio project as a dependency. So the content
+of `android/build.gradle` will be the following:
+
+    apply plugin: 'java-library'
+    apply plugin: 'kotlin-platform-jvm'
+
+    // Specify a group and a version of the library to access it in Android Studio.
+    group = 'org.greeting'
+    version = 1.0
+
+    dependencies {
+        // Specify Kotlin/JVM stdlib dependency.
+        implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+
+        // Specify dependency on a common project for Kotlin multiplatform build.
+        expectedBy project(':common')
+    }
+
+#### 2.3 iOS subproject
+
+As well as `android`, this project contains platform-dependent implementations of `expect`-declarations:
+
+    // greeting/ios/src/main/kotlin/ios.kt
+    package org.greeting
+
+    actual class Platform actual constructor() {
+        actual val platform: String = "iOS"
+    }
+
+We build this project into an Objective-C framework using Kotlin/Native compiler. To do this, declare a framework in
+`ios/build.gradle` and add an `expectedBy` dependency in the same manner as in the Android project:
+
+    apply plugin: 'konan'
+
+    konanArtifacts {
+        // Declare building into a framework.
+        framework('ios-greeting') {
+            // The multiplatform support is disabled by default.
+            enableMultiplatform true
+        }
+    }
+
+    dependencies {
+        // Specify dependency on a common project for Kotlin multiplatform build
+        expectedBy project(':common')
+    }
+
+### 3. Android application
+
+Now we can create an Android application which will use the library we implemented on the previous step. Open Android
+Studio and create a new project in the `androidApp` directory. Android Studio generates all necessary files and
+directories so we need only add a dependency on our library. There are only 2 actions we have to do:
+
+1. Add dependency on the library. To do this just open `androidApp/app/build.gradle` and add the following snippet in
+the `dependencies` script block:
+
+    ```
+    implementation 'org.greeting:android:1.0'
+    implementation 'org.greeting:common:1.0'
```

Comment:
Is it possible to define the name of `android` and `common` in the specific `build.gradle` files?
For me it is clear that it is the name of the directory but maybe not for beginners...

Reply:
done